src/corelibs/U2Gui/src/options_panel/GroupOptionsWidget.h

```diff
@@ -69,7 +69,7 @@ class U2GUI_EXPORT GroupOptionsWidget : public QWidget {
     QVBoxLayout *mainLayout;
 
     static const int TITLE_HEIGHT = 30;
-    static const int MIN_WIDGET_WIDTH = 300;
+    static const int MIN_WIDGET_WIDTH = 440;
```

Comment:
This is big problem here.
I would like to ask you to remove this change from the patch and even can help you to implement this feature correctly if you need help.

Here are the details:
- What your patch does: makes all kinds of tabs in all kinds of Option Panels bigger: assembly/sequence/tree/msa/sanger/etc...
- What is the correct solution:  allow every individual tab to override the defaults. So only when your plugin is loaded and only when user clicks the plugin's tab of the option panel grows in size.

This is how it is implemented in other tools and as I've said I can help you with this impl if you need help.

Reply:
Removed, postpone it for now